src/xx-info

```diff
@@ -337,6 +340,14 @@ case "$TARGETARCH" in
     XX_RHEL_ARCH="mips64el"
     XX_TRIPLE="mips64el${vendor}-linux-${XX_LIBC}abi64"
     ;;
+  "loongarch64")
+    XX_MARCH="loongarch64"
+    XX_DEBIAN_ARCH="loongarch64"
```

Comment:
Looks like debian name is `loong64` ?

Reply:
https://github.com/guillemj/dpkg/blob/main/data/cputable#L28

Reply:
This is the same as the above problem, I will add

Reply:
Or you can directly add support for ABI2.0, because ABI1.0 has not been officially supported by Debian

Reply:
Or, we can directly add support for ABI2.0, because Debian has not officially supported ABI1.0